tests/Ticket/1325TestCase.php

```diff
@@ -49,7 +51,6 @@ public function testShouldInsertWithoutAlias()
             ->from('Ticket_1325_TableName_NoAlias')
             ->fetchOne(array(), Doctrine_Core::HYDRATE_ARRAY);
 
-        $now = time();
         $time = strtotime($res['event_date']);
```

Comment:
Can't we align the two tests with the same check? In the "aliased" test, it is only checking that the eventDate column is set and greater than zero

Reply:
Sure, sorry, I didn't go to the details, just have seen the unused variables and removed them. Pr is updated accordingly.